test/json/builder.spec.js

```diff
@@ -63,11 +63,25 @@
             Items: null
           })).to.equal('{}');
         });
-        return it('ignores undefined', function() {
+        it('ignores undefined', function() {
           return expect(build(rules, {
             Items: void 0
           })).to.equal('{}');
         });
+        it('builds document types', function() {
```

Comment:
```suggestion
        return it('builds document types', function() {
```

Reply:
Return statement is not necessary. I remove them as I touch the test files. They are generated from the CoffeeScript.